agr_api/src/test/java/org/alliancegenome/api/tests/integration/DiseaseIT.java

```diff
@@ -706,7 +706,7 @@ public void checkDiseaseAnnotationNonDuplicated() {
     @Test
     // Test Sox9 from MGI for disease via experiment records
     public void checkDiseaseRibbonHeader() {
-        DiseaseRibbonSummary summary = diseaseService.getDiseaseRibbonSummary(List.of("MGI:98297"));
+        DiseaseRibbonSummary summary = diseaseService.getDiseaseRibbonSummary(List.of("MGI:98297"), null);
```

Comment:
Typically its good to create another method that supports the absence of the parameter, that then calls the method with null.

Reply:
Completely, agree. Will do.

Reply:
In this case, there is no consumer for this method other than a unit test method.